src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -50,6 +50,12 @@ ucs_config_field_t uct_ib_mlx5_iface_config_table[] = {
      ucs_offsetof(uct_ib_mlx5_iface_config_t, mmio_mode),
      UCS_CONFIG_TYPE_ENUM(uct_ib_mlx5_mmio_modes)},
 
+    {"AR_ENABLE", "n",
```

Comment:
let's make it 'auto' for compatibility

Reply:
`TERNARY` type doesn't have `auto`, do you mean to add it?

Reply:
sorry, i mean 'try'